LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
```

Comment:
So no more license at all? :(

Reply:
We can't have MIT license if we're including code that's linked to IP. Might be ok, but being a bit cautious

Reply:
Sure, I understand. Would be great to decide for a license because otherwise people won't know if and how to use this.  
/cc @arpee